src/test/resources/pages/studentCourseJoinConfirmationHTML.html

```diff
@@ -106,7 +106,7 @@
                      .
                      <br/>
                                           If this is not you please
-                     <a href="/_ah/logout?continue=%2Fpage%2FstudentCourseJoinAuthenticated%3Fkey%3D{*}%26next%3D%2Fpage%2FstudentHomePage">
```

Comment:
This handles the difference in the urls generated in dev and staging servers. in staging the entire app url is shown